src/components/Facility/Investigations/Reports/utils.tsx

```diff
@@ -1,8 +1,10 @@
-import { memoize, chain, findIndex } from "lodash-es";
-import { InvestigationResponse } from "./types";
+import * as _ from "lodash-es";
+import { findIndex, memoize } from "lodash-es";
```

Comment:
@sainak Why are we importing twice from lodash. Can we solve this. Let's not import *